msteams-platform/graph-permissions/resource-specific-consent.md

```diff
@@ -0,0 +1,172 @@
+|TeamSettings.Edit.Group|Update the settings for this team.|
+|ChannelSettings.Read.Group|Get the channel names, channel descriptions, and channel settings for this team​.|
+|ChannelSettings.Edit.Group|Update the channel names, channel descriptions, and channel settings for this team.​|
+|Channel.Create.Group|Create channels in this team.​|
+|Channel.Delete.Group|Delete channels in this team.​|
+|ChannelMessage.Read.Group |Get this team's channel messages.​|
+|TeamsApp.Read.Group|Get a list of this team's installed apps.|
+|TeamsTab.Read.Group|Get a list of this team's tabs.|
+|TeamsTab.Create.Group|Create tabs in this team.​|
+|TeamsTab.Edit.Group|Update this team's tabs.​|
+|TeamsTab.Delete.Group|Delete this team's tabs.​|
+|Member.Read.Group|Get this team's members.​|
+|Owner.Read.Group|Get this team's owners.​|
+|Member.ReadWrite.Group|Add and remove members from this team.​|
+|Owner.ReadWrite.Group|Add and remove owners from this team.​|
+|File.Read.Group|Open a file,get all text, and close the file in this team.|
+|File.Create.Group|Create or open a file in this team. Note: if the file already exists, its contents will be overwritten.|
+|File.Edit.Group|Update the name of a specified file in this team.|
+|File.Delete.Group|Delete a specified file in this team.|
+
+>[!NOTE]
+>Resource specific permissions are only available to Teams apps installed on the Teams client and are currently not part of Azure AD.
+
+### **Enabling RSC in your application** 
+
+## 1. Ensure that the Tenant admin has configured [group owner consent](/azure/active-directory/manage-apps/configure-user-consent#configure-group-owner-consent-to-apps-accessing-group-data) settings in Azure Active Directory (Azure AD)
+
+- **Disable or enable group owner consent from the Azure portal**  
+
+> [!div class="checklist"]
+>
+>- Sign in to the [Azure portal](https://portal.azure.com) as a [Global Administrator/Company Administrator](/azure/active-directory/users-groups-roles/directory-assign-admin-roles.md#global-administrator--company-administrator).  
+ > - Select **Azure Active Directory** =>**Enterprise applications** =>**User settings**.
+> - Enable, disable, or limit user consent with the control labeled **Users can consent to apps accessing company data for the groups they own** (This capability is enabled by default)
+
+![azure rsc configuration](../assets/images/azure-rsc-configuration.svg)
+
+| Value | Description|
+|--- | --- |
+|Yes | Enable group-specific consent for all group owners.|
+|No |Disable group-specific consent for all users.| 
+|Limited | Enable group-specific consent for members of a selected group.|
+
+- **Disable or enable group owner consent using PowerShell**. Please follow the steps outlined in  the [Configure group owner consent using PowerShell](/azure/active-directory/manage-apps/configure-user-consent#configure-group-owner-consent-using-powershell) documentation.
+
+## 2.  [Register your app using the Azure portal](/graph/auth-register-app-v2)
```

Comment:
Should there be more text in this section? Key point is "register it like any other Graph app using application permissions".